apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/slide-control.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import { times } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { SVG, Circle } from '@wordpress/primitives';
+import { Button } from '@wordpress/components';
+
+/*
+ * Copied from the Guide Component: https://github.com/WordPress/gutenberg/blob/4bbfbc13603d28fcc45368c529954164bf8581de/packages/components/src/guide/page-control.js#L3
+ * Note that it is styled by .components-guide-* styling that already exists from using the Guide package elsewhere
+ * TODO: make this into WP package?
+ */
```

Comment:
I took this code from Guide component's [PageControl](https://github.com/WordPress/gutenberg/blob/4bbfbc13603d28fcc45368c529954164bf8581de/packages/components/src/guide/page-control.js#L16), which is not publicly accesible.  Would it be worthwhile to make it it's own Gutenberg component?  It's pretty handy! :)

Reply:
> Would it be worthwhile to make it it's own Gutenberg component? It's pretty handy! :)

Definitely! Not a blocker for your work but I think it'd be a nice little Gutenberg contribution